npm/simple-mock.js

```diff
@@ -0,0 +1,914 @@
+    var allowUnmocked = { allowUnmocked: true };
+
+    nock('https://postman-echo.com', allowUnmocked)
+        .persist()
+        // .log(console.log)
+        .get('/gzip')
+        .reply(function () {
+            return [
+                200,
+                {
+                    gzipped: true,
+                    headers: {
+                        'x-forwarded-proto': 'https',
+                        host: 'postman-echo.com',
+                        accept: '*/*',
+                        'accept-encoding': 'gzip, deflate',
+                        'cache-control': 'no-cache',
+                        cookie:
+                        // eslint-disable-next-line max-len
+                        'sails.sid=s%3AzVZKZEAc1IvL61qZyqrR2MSRO49mI9bc.iVKWR9D0pzmWZf158wNUp6thKfAHKQRU4JTUxnKFurc; foo=bar',
+                        'postman-token': '1c51c9ac-e4b4-40c0-a821-9317d70d9dc1',
+                        'user-agent': 'PostmanRuntime/7.6.0',
+                        'x-forwarded-port': '443'
+                    },
+                    method: 'GET'
+                },
+                this.req.headers
+            ];
+        });
+}
+
+function mockRedirects () {
+    var allowUnmocked = { allowUnmocked: true };
+
+    nock(/https:\/\/.*.com\//, allowUnmocked)
+        .persist()
+        // .log(console.log)
+        .get(/\/redirect-to.*/)
+        // .query(true)
+        .reply(function () {
+            return [302, {
+                args: {},
+                headers: {
+                    'x-forwarded-proto': 'https',
+                    host: 'postman-echo.com',
+                    accept: '*/*',
+                    'accept-encoding': 'gzip, deflate',
+                    'cache-control': 'no-cache',
+                    cookie:
+                    // eslint-disable-next-line max-len
+                    'sails.sid=s%3Amz8FY8zkbUSVOHUm1vV6dhz9yi7vpanU.zbJ%2FdhTOwNu%2BKelkJqM8LF1rODiONamyejECgbnEmig; foo=bar',
+                    'postman-token': 'c4467e84-85cd-4ea1-bfa4-5d0ddf855025',
+                    referer: 'https://postman-echo.com/redirect-to?url=https://postman-echo.com/get',
+                    'user-agent': 'PostmanRuntime/7.6.0',
+                    'x-forwarded-port': '443'
+                },
+                url: 'https://postman-echo.com/get'
+            }, this.req.headers];
+        });
+}
+
+function mockDELAYrequest () {
+    nock('https://postman-echo.com')
+        .persist()
+        .get(/\/delay.*/)
+        .query(true)
+        .reply(200, function (uri) {
+            var reply = { };
+
+            reply.delay = uri.substr(-1);
+
+            return reply;
+        }, {
+            'Content-Type': 'application/json; charset=utf-8',
+            Server: 'nginx',
+            'set-cookie':
+            // eslint-disable-next-line max-len
+            'sails.sid=s%3A3QB9NKdf_MZjNJaMXNPE-snq-154akMU.c1AJ4OkwgGsyIrGYNWZhuXMaddwrMHidJTWJQGOINlE; Path=/; HttpOnly',
```

Comment:
How was this value generated?

Reply:
This header is copied from a postman app console, from a response header of the corresponding request.